generators/chipyard/src/main/scala/config/RocketConfigs.scala

```diff
@@ -21,6 +21,13 @@ class TinyRocketConfig extends Config(
   new freechips.rocketchip.subsystem.With1TinyCore ++             // single tiny rocket-core
   new chipyard.config.AbstractConfig)
 
+class FFTRocketConfig extends Config(
+  new fftgenerator.WithFFTNumPoints(8) ++
+  new fftgenerator.WithFFTBaseAddr(0x2000) ++
+  new fftgenerator.WithFFTGenerator ++
```

Comment:
I think this should all be added with a single config option. I.e.

```
new fftgenerator.WithFFTGenerator(baseAddr=addr, params=potentialOverrideParams)++
```

At minimum, the `FFTNumPoints` is really set by the `FixedTailParams` so instead of creating a new KV pair (`FFTNumPoints`), instead you should just modify the `FixedTailParams` in the config fragment. See the other PR for an example